tgui/packages/tgui/interfaces/ServerControl.tsx

```diff
@@ -0,0 +1,150 @@
+    return (
+      <Window width={575} height={400}>
+        <Window.Content>
+          <NoticeBox textAlign="center" danger>
+            Not connected to a Server. Please sync one using a multitool.
+          </NoticeBox>
+        </Window.Content>
+      </Window>
+    );
+  }
+  return (
+    <Window width={575} height={400} scrollable fill>
+      <Window.Content scrollable>
+        {!servers ? (
+          <NoticeBox mt={2} info>
+            No servers found.
+          </NoticeBox>
+        ) : (
+          <Section>
+            <Table cellpadding="3" textAlign="center">
+              <Table.Row header>
+                <Table.Cell>Research Servers</Table.Cell>
+              </Table.Row>
+              {servers.map((server) => (
+                <>
+                  <Table.Row header key={server} />
+                  <Table.Cell> {server.server_name}</Table.Cell>
+                  <Button
+                    mt={1}
+                    tooltip={server.server_details}
+                    color={server.server_disabled ? 'bad' : 'good'}
+                    content={server.server_disabled ? 'Offline' : 'Online'}
+                    fluid
+                    textAlign="center"
+                    onClick={() =>
+                      act('lockdown_server', {
+                        selected_server: server.server_ref,
+                      })
+                    }
+                  />
+                </>
+              ))}
+            </Table>
+          </Section>
+        )}
+
+        {!consoles ? (
+          <NoticeBox mt={2} info>
+            No consoles found.
+          </NoticeBox>
+        ) : (
+          <Section align="right">
+            <Table cellpadding="3" textAlign="center">
+              <Table.Row header>
+                <Table.Cell>Research Consoles</Table.Cell>
+              </Table.Row>
+              {consoles.map((console) => (
+                <>
+                  <Table.Row header key={console} />
+                  <Table.Cell>
+                    {' '}
+                    {console.console_name} - Location:{' '}
+                    {console.console_location}{' '}
+                  </Table.Cell>
+                  <Button
+                    mt={1}
+                    color={console.console_locked ? 'bad' : 'good'}
+                    content={console.console_locked ? 'LOCKED' : 'UNLOCKED'}
+                    fluid
+                    textAlign="center"
+                    onClick={() =>
+                      act('lock_console', {
+                        selected_console: console.console_ref,
+                      })
+                    }
+                  />
+                </>
+              ))}
+            </Table>
+          </Section>
+        )}
+
+        <Collapsible title="Research History">
+          {!logs.length ? (
+            <NoticeBox mt={2} info>
+              No history found.
+            </NoticeBox>
+          ) : (
+            <Table>
```

Comment:
please also wrap this in a section, the white on empty gray is still present

Reply:
![image](https://user-images.githubusercontent.com/53777086/215306288-caadd398-7b04-422b-8077-156710e3d667.png)